### DISCUSSION TITLE
Robosats Desktop 0.7: tor EACCES error

### DISCUSSION THREAD

**Discussion by rolznz** [RoboSats/robosats#1752] [Q&A]
**Describe the bug**
Running the desktop app seems to have an access error which breaks the app.

```
robosats-desktop-v0.7.0-linux-x64$ ./Robosats 
A JavaScript error occurred in the main process
Uncaught Exception:
Error: spawn /ommitted/robosats-desktop-v0.7.0-linux-x64/resources/app/tor/tor-linux/tor/tor EACCES
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

**To Reproduce**
1. Download the latest desktop app
2. Extract it
3. chmod +x the filename
4. Run it (see error above)

**Expected behavior**
No error at startup

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.5 LTS
 - Version robosats-desktop-v0.7.0-linux-x64

**Additional context**
First time trying the desktop app

**Comment by KoalaSat** [comment]
hi @rolznz I tested it and it worked for me, is it possible that the reason is you extract the file with permissions not open to your ubuntu user?
